src/calculateRentalCost.js

```diff
@@ -28,6 +28,18 @@
  */
 
 function calculateRentalCost(days) {
+  const rent = 40;
+  let sum = 0;
+
+  if (days >= 7) {
+    sum = (rent * days) - 50;
+  } else if (days < 3) {
+    sum = (rent * days);
+  } else if (days >= 3 && days < 7) {
+    sum = (days * rent) - 20;
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean